src/Build/BuildCheck/OM/ProjectImportedCheckData.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Build.Experimental.BuildCheck;
+
+namespace Microsoft.Build.Experimental.BuildCheck;
+
+public class ProjectImportedCheckData : CheckData
+{
+    public ProjectImportedCheckData(string importedProjectFile, string projectFilePath, int? projectConfigurationId)
```

Comment:
Should be internal - there is no need for it to be public and it'd limit our freedom to change in future